src/substrate/filters/enricher.ts

```diff
@@ -38,25 +38,31 @@ export async function Enrich(
         const [ sessionIndex ] = event.data as unknown as [ SessionIndex ] & Codec
         const validators = await api.derive.staking.validators();
         const currentEra = await api.query.staking.currentEra<Option<EraIndex>>();
-        let exposure : Array<Exposure>
         let active : Array<ValidatorId>
         let waiting : Array<ValidatorId>
-        // erasStakers(EraIndex, AccountId): Exposure -> api.query.staking.erasStakers
-        if (validators && currentEra.isSome) { // if currentEra isn't empty
+        // erasStakers(EraIndex, AccountId): Exposure -> api.query.staking.erasStakers // KUSAMA
+        // stakers(AccountId): Exposure -> api.query.staking.stakers // EDGEWARE
+        const stakersCall = (api.query.staking.stakers)
+          ? api.query.staking.stakers
+          : api.query.staking.erasStakers;
+        const stakersCallArgs = (account) => (api.query.staking.stakers)
+        ? account
+        : [+currentEra, account];
+        let activeExposures : {[key: string]: any} = {}
+        if (validators && currentEra) { // if currentEra isn't empty
           active = validators.validators;
           waiting = validators.nextElected;
           active.forEach(async (validator) => {
```

Comment:
Since this is an array call wrapping an async, you need to change this to an `await Promise.all(active.map(async (validator) => ...))`.